File: python/3.KidsWithTheGreatestNumberOfCandies.py
def kidsWithCandies(candies, extraCandies):
    kidsWithExtraCandies = []
    maxValue = max(candies)
    for candy in candies:
        kidsWithExtraCandies.append(candy + extraCandies >= maxValue)
    return kidsWithExtraCandies


print(kidsWithCandies([2, 3, 5, 1, 3], 3))
print(kidsWithCandies([4, 2, 1, 1, 2], 1))
print(kidsWithCandies([12, 1, 12], 1))

File: javascript/3.KidsWithTheGreatestNumberOfCandies.js
function kidsWithCandies(candies, extraCandies) {
    let kidsWithExtraCandies = []
    const maxValue = Math.max(...candies)
    candies.forEach((candy) => kidsWithExtraCandies.push(candy + extraCandies >= maxValue))
    return kidsWithExtraCandies
}

console.log(kidsWithCandies([2, 3, 5, 1, 3], 3))
console.log(kidsWithCandies([4, 2, 1, 1, 2], 1))
console.log(kidsWithCandies([12, 1, 12], 10))
